index.html

```diff
@@ -369,25 +373,14 @@ <h3>
         <tbody>
           <tr>
             <td>
-              <dfn>[[\InstanceList]]</dfn>
+              <dfn>[[\ActiveLocks]]</dfn>
             </td>
             <td>
               The empty <a>ordered set</a>.
             </td>
             <td>
-              A <a>ordered set</a> of {{WakeLock}} instances.
-            </td>
-          </tr>
-          <tr>
-            <td>
-              <dfn>[[\RequestCounter]]</dfn>
-            </td>
-            <td>
-              0.
-            </td>
-            <td>
-              The value indicates the amount of current requests for the
-              <a>wake lock type</a>.
+              A <a>ordered set</a> of {{Promise}}s, representing active
```

Comment:
An ordered set. Does it actually need to be ordered? This also happens to fix #192.

Reply:
No, unordered is fine... just need to check how to change that :)

Reply:
Seems to be "list"